src/main/config/config.c

```diff
@@ -470,7 +470,7 @@ static void resetConf(void)
     masterConfig.escAndServoConfig.maxthrottle = 2000;
     masterConfig.motor_pwm_rate = 32000;
     masterConfig.looptime = 2000;
-//    currentProfile->pidController = 3;
+    currentProfile->pidController = 3;
```

Comment:
i had to read that `#ifdef` stuff twice before i realised that too @ledvinap  :smile: